package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "yoast-comment-hacks",
+  "name": "comment-hacks",
   "description": "Development files for the Comment Hacks plugin",
   "license": "GPL-2.0+",
   "plugin": {
     "glotpress": "https://translate.wordpress.org",
-    "textdomain": "yoast-comment-hacks"
+    "textdomain": "comment-hacks"
   },
   "pot": {
     "reportmsgidbugsto": "https://github.com/emilia-capital/comment-hacks/issues",
-    "languageteam": "Yoast Translate <translations@yoast.com>",
-    "lasttranslator": "Yoast Translate Team <translations@yoast.com>"
+    "languageteam": "Emilia Projects Translate <translations@emilia.capital>",
```

Comment:
Here I think this is fine, we should make a task to create this email alias though.

Reply:
Added a task in https://github.com/Emilia-Capital/comment-hacks/issues/116